api/prisma/schema.prisma

```diff
@@ -71,9 +71,9 @@ model YDocSnapshot {
     id        String   @id
     createdAt DateTime @default(now())
     message   String?
-    yDocBlob  Bytes?
-    repo      Repo?    @relation(fields: [repoId], references: [id])
-    repoId    String?
+    yDocBlob  Bytes
+    repo      Repo    @relation(fields: [repoId], references: [id])
+    repoId    String
 }
```

Comment:
You need to generate a new DB migration to reflect this change.

Reply:
[bb35923](https://github.com/codepod-io/codepod/pull/462/commits/bb359237593f4a168b3b5151d3888630f9c72efa) addes the prisma migration.